assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -43,6 +43,11 @@ export default {
         });
       }
 
+      if (this.siteSettings.enable_experimental_hashtag_autocomplete) {
+        api.registerHashtagSearchParam("category", "chat-composer", 100);
+        api.registerHashtagSearchParam("tag", "chat-composer", 50);
+      }
```

Comment:
Related to https://github.com/discourse/discourse/pull/18592/files#r998915006

Do you think we should use separate api block like `withPluginApi("0.14.1"`

Reply:
I don’t want to bump it yet since it’s experimental I’ll do that once it’s more solid